backend/constant/common.go

```diff
@@ -116,6 +116,7 @@ var WebUrlMap = map[string]struct{}{
 	"/xpack/gpu":               {},
 	"/xpack/alert/dashboard":   {},
 	"/xpack/alert/log":         {},
+	"/xpack/alert/setting":     {},
 	"/xpack/setting":           {},
 }
 
```

Comment:
The code has no irregularities nor suggestions to optimize. Everything is done correctly up until now. The "WebUrlMap" struct looks complete and ready to store and retrieve URLs using keys (in this case, the path segments). There appears to be no issue with this specific part of the code.

As mentioned earlier, if we were checking the current date instead of 2021-09-01, then it would look like this:

```
Current date: [current_date], Knowledge cutoff: <current_date>-<knowledge_cutoff>, Current year: [[year]],
Currently, knowledge about web programming from that time frame was used.
```